src/main/java/net/pms/dlna/DLNAResource.java

```diff
@@ -2257,6 +2257,8 @@ private String getDlnaOrgPnFlags(RendererConfiguration mediaRenderer, int locali
 				} else if (mime.equals("video/vnd.dlna.mpeg-tts")) {
 					// patters - on Sony BDP m2ts clips aren't listed without this
 					dlnaOrgPnFlags = "DLNA.ORG_PN=" + getMPEG_TS_EULocalizedValue(localizationValue, media.isHDVideo());
+				} else if (mime.equals(MP4_TYPEMIME)) {
+					dlnaOrgPnFlags = "*";
```

Comment:
The converted file plays, just not while streaming. You said you use it all the time, so can you let me know how the following is different to what you use?

```
"C:\Program Files (x86)\Universal Media Server\win32\ffmpeg64.exe" -y -loglevel info -i D:\Klaus\Unsorted\Unwatched\Adventure.Time.With.Finn.And.Jake.S02E18.1080p.BluRay.x264-DEiMOS.mkv -bufsize 30000k -maxrate 29000k -crf 23 -ab 320k -ar 48000 -c:a aac -c:v libx264 -tune zerolatency -preset ultrafast -level 31 -pix_fmt yuv420p -f mp4 -movflags frag_keyframe+empty_moov file.mp4
```